$NetBSD: patch-aa,v 1.1.1.1 2007/07/29 21:33:42 seb Exp $

The source to the configure script is not in the distribution
tarball.

--- configure.orig	2006-05-21 16:30:10.000000000 +0000
+++ configure
@@ -3927,7 +3927,7 @@ echo "$as_me: error: Link with \"-lz\" f
 fi
 
 if test -d "$WHERE_FTPATH"; then
-	if test ! -f "$WHERE_FTPATH/lib/ftlib.h"; then
+	if test ! -f "$WHERE_FTPATH/include/flow-tools/ftlib.h"; then
 		{ { echo "$as_me:$LINENO: error: ftlib.h file not found in flow-tools directory '$WHERE_FTPATH'. Use --with-ftpath=PATH" >&5
 echo "$as_me: error: ftlib.h file not found in flow-tools directory '$WHERE_FTPATH'. Use --with-ftpath=PATH" >&2;}
    { (exit 1); exit 1; }; }
@@ -3937,7 +3937,7 @@ echo "$as_me: error: ftlib.h file not fo
 echo "$as_me: error: libft.a not found in flow-tools directory '$WHERE_FTPATH'. Build flow tools first" >&2;}
    { (exit 1); exit 1; }; }
 	fi
-	FT_INCLUDES="-I$WHERE_FTPATH/src -I$WHERE_FTPATH/lib"
+	FT_INCLUDES="-I$WHERE_FTPATH/include/flow-tools"
 	FT_LDFLAGS="-L$WHERE_FTPATH/lib"
 else
 	{ { echo "$as_me:$LINENO: error: flow-tools directory '$WHERE_FTPATH' does not exists. Use --with-ftpath=PATH" >&5
